tests.py

```diff
@@ -162,18 +163,19 @@ def test_new_bookmark_read_is_unread(self):
         res = self.app.get(
             '/api/v1/bookmarks/{}'.format(bookmark_id),
             content_type='application/json',
-            data=json.dumps({'read': True}),
+            data=json.dumps({'read': read}),
```

Comment:
It seems it's going to be fixed by this patch: https://github.com/heutagogy/heutagogy-backend/pull/55